.github/actions/setup-node/action.yaml

```
@@ -13,7 +13,7 @@ runs:
     - name: Setup Node
       uses: buildjet/setup-node@v3
       with:
-        node-version: 16.20.1
+        node-version: 18.17.0
```

```suggestion
        node-version: 18.x
```
As we are installing the latest 18.x release in Dockerfile should we also do this in ci?
@kylecarbs for a 2nd thought.

---

Why not? The docker file is for the workspace and the CI should use the same version as well.